src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -287,10 +301,19 @@
             "CurrentMaxRate",
             "OverrunCount"
         ],
-        "Channel": ["Lineup", "CurrentChannel"],
-        "Media Playback": ["SampledPosition"],
-        "Application Launcher": ["CurrentApp"],
-        "Application Basic": ["Application"],
+        "Channel": [
+            "Lineup",
+            "CurrentChannel"
+        ],
+        "Media Playback": [
```

Comment:
i m not against fixing the indentation but i would favor separate PR for this

Reply:
Any change in indentation was not intentional.  Maybe it's an autoformatter in VS Code?  I will investigate.